product-features/liquidity-providers/faqs.md

```diff
@@ -0,0 +1,79 @@
+
+Also know as "Pool Tokens", these tokens are issued to liquidity providers to represent their share of the liquidity pool. The total supply of LP tokens represents the 100% of the pool's funds.
+
+When users add liquidity to a pool, they receive LP tokens as proof of ownership. These tokens entitle them to a proportional share of the pooled assets and a portion of the fees generated by trades (swaps) within the pool.
+
+When liquidity is removed, the user transfers LP tokens back to the protocol. This determines how much of the pool's assets are returned to the user, along with their share of the transaction fees accrued during the time their liquidity was provided.
+
+</details>
+
+<details>
+
+<summary>How does the fee rebate benefit liquidity providers?</summary>
+
+The fee rebate is automatically reinvested into the pool, increasing the overall value of the pool. Since liquidity providers (LPs) hold a share of the pool, their holdings grow in value over time. However, these rewards can only be claimed when LP tokens (representing their share of liquidity) are withdrawn from the pool.
+
+</details>
+
+<details>
+
+<summary>How are fees transferred to the liquidity provider's wallet?</summary>
+
+The fees are not directly transferred to the liquidity provider's (LP) wallet. Instead, the swap fees allocated to LPs are reinvested into the liquidity pool. By holding LP tokens, liquidity providers accumulate their share of the fees over time. These accrued fees become available when they withdraw funds from the pool (i.e., when they remove liquidity). At that point, LP tokens are transferred back to the protocol, and in return, the provider receives their corresponding share of the pool's funds, including the accumulated fees.
+
+</details>
+
+<details>
+
+<summary>Where can I see the fees I've gained so far as a liquidity provider?</summary>
+
+You can view your accrued fees in the "My Liquidity" panel. To access it, navigate to the Swap -> Pool tab and select your pool of interest from the list. You'll also see a summarized version above the pool list.
+
+In this panel, the "Pooled" amount reflects your total token holdings in the liquidity pool, which includes both your initial deposit and any fees you've accrued. Over time, this amount increases as more fees are added. The "Indicative Value" shows the USD equivalent of your holdings, which may fluctuate due to price changes of the pool's assets, but still provides a useful reference for tracking your gains.
+
+</details>
+
+<details>
+
+<summary>How is liquidity provision related to farming?</summary>
+
+Liquidity providers can stake or lock up their LP tokens for a fixed period to earn additional rewards. These rewards are separate from the earnings generated through liquidity provision, that come from swap operations fees (trading fees). This process is known as Yield Farming, or simply 'Farming'. For more details, explore the [ALEX Farming](../farm.md) feature.
```

Comment:
Liquidity providers can stake or lock up their LP tokens for a fixed period(a selected number of ALEX Cycles) to earn additional rewards. These rewards are separate from the earnings generated through liquidity provision, that come from swap operations fees (trading fees). This process is known as Yield Farming, or simply 'Farming'. For more details, explore the [ALEX Farming](../farm.md) feature.